site/content/docs/5.3/examples/download-app/download-app.css

```diff
@@ -0,0 +1,245 @@
+.btn-secondary {
+  background-color: transparent;
+}
+
+.border-translucent {
+  --bs-border-color: var(--bs-border-color-translucent);
+}
+
+@media (max-width: 479.98px) {
+  h3,
+  .h3 {
+    font-size: 1.125rem;
+    line-height: 1.111111;
+    letter-spacing: -.0125rem;
+  }
+}
+
+.icons-sm {
+  width: 2.5rem;
+  height: 2.5rem;
+}
+
+.icons-md {
+  width: 3.125rem;
+  height: 3.125rem;
+}
+
+.icons-lg {
+  width: 4.375rem;
+  height: 4.375rem;
+}
+
+@media (min-width: 768px) {
+  .icons-sm {
+    width: 3.125rem;
+    height: 3.125rem;
+  }
+
+  .icons-md {
+    width: 3.75rem;
+    height: 3.75rem;
+  }
+
+  .icons-lg {
+    width: 6.25rem;
+    height: 6.25rem;
+  }
+}
+
+.out-of-grid {
+  position: static;
+}
+
+.out-of-grid img {
+  width: 100%;
+  min-width: 100%;
+  min-height: 165px;
+}
+
+@media (min-width: 1024px) {
+  .out-of-grid {
+    position: absolute;
+  }
+
+  .out-of-grid img {
+    width: auto;
+  }
+}
+
+iframe {
+  min-height: 10.625rem;
+}
+
+footer .nav-link:hover {
+  text-decoration: underline;
+}
+
+.images img {
+  border-radius: 12% / 5%;
+  box-shadow: 10px 15px 50px -5px rgba(var(--bs-link-color-rgb), .2);
+  transition: transform .3s ease-in-out;
+  transform-origin: center;
+}
+
+@media (min-width: 768px) {
+  .images.active > :hover img {
+    transform: scale(1.2);
+  }
+}
+
+@media (prefers-reduced-motion) {
+  .images.active > :hover img {
+    transform: none;
+  }
+}
+
+@media (min-width: 768px) {
+  .images > .col {
+    transform: translateY(-50px);
+  }
+
+  .images.active > .col {
+    animation: enter .75s 4 alternate, enter-smooth 1.25s 1 forwards ease-in-out;
+  }
+}
+
+@media (prefers-reduced-motion) {
+  .images > .col {
+    transform: none;
+  }
+
+  .images.active > .col {
+    animation: none;
+  }
+}
+
+@keyframes enter {
+  0% {
+    transform: translateY(-50px);
+    animation-timing-function: cubic-bezier(.5, 0, 1, .5);
+  }
+  100% {
+    transform: translateY(0);
+    animation-timing-function: cubic-bezier(0, .5, .5, 1);
+  }
+}
+
+@keyframes enter-smooth {
+  0% {
+    transform: translateY(-50px);
+  }
+  100% {
+    transform: translateY(0);
+  }
+}
+
+.images.active > :nth-child(1) {
+  animation-delay: 0ms, 3000ms;
+}
+
+.images.active > :nth-child(2) {
+  animation-delay: 100ms, 3100ms;
+}
+
+.images.active > :nth-child(3) {
+  animation-delay: 200ms, 3200ms;
+}
+
+.images.active > :nth-child(4) {
+  animation-delay: 300ms, 3300ms;
+}
+
+.images.active > :nth-child(5) {
+  animation-delay: 400ms, 3400ms;
+}
+
+.images.active > :nth-child(6) {
+  animation-delay: 500ms, 3500ms;
+}
+
+.ticks > li {
+  opacity: 0;
+}
+
+.ticks.active > li {
+  animation: enter-start-smooth .75s 1 forwards ease-in-out;
+}
+
+@media (prefers-reduced-motion) {
```

Comment:
Just saw this article: https://nicolas-hoizey.com/articles/2023/04/07/running-css-animations-only-if-both-the-device-and-the-user-allow-it/

I think it could be a good way to improve a11y and general performance. Should we give it a try?

Reply:
Yes it may be too soon to use it. I think we can wait for a better support but I will it will be a really interesting feature soon...